.travis.yml

```diff
@@ -3,12 +3,19 @@ language: php
 php:
   - 7.0
   - 7.1
+  - 7.2
+  - 7.3
+  - 7.4
```

Comment:
HubSpot refuses many connection and that is why we specified only two version

Reply:
I've updated it so that only the lowest and highest versions run integration tests :)